src/app/components/GearInput/GearInput.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import GearInput from './GearInput';
+
+export default {
+  title: 'Component/GearInput',
+  component: GearInput,
+  parameters: {
+    backgrounds: {
+      default: 'dark',
+    },
+    layout: '!padded',
```

Comment:
This is [default and would be written without "!"](https://storybook.js.org/docs/react/configure/story-layout#global-layout)

Reply:
Thank you for reviewing. Storybook adds a default padding, that I didn't want to see. This is just a quick workaround.